roles/DCOS.agent_windows/vars/main.yml

```diff
@@ -0,0 +1,13 @@
+# DCOS params:
+dcos_version_specifier: "{{ dcos['version'] }}"

```

Comment:
not used anymore if the `url_to_mesosbin` is changed

Reply:
removed the variable `dcos_version_specifier` and using `{{ dcos['version'] }}` going forward